src/types/index.ts

```diff
@@ -340,6 +365,34 @@ export type PaymentRequestButtonElementComponent = FunctionComponent<
   PaymentRequestButtonElementProps
 >;
 
+export interface ShippingAddressElementProps extends ElementProps {
+  /**
+   * An object containing [Element configuration options](https://stripe.com/docs/js/elements_object/create_element?type=afterpayClearpayMessage).
+   */
+  options?: stripeJs.StripeShippingAddressOptions;
```

Comment:
Not introduced in this PR, but should this have been `StripeShippingAddressElementOptions` for consistency with the other Elements?
```suggestion
  options?: stripeJs.StripeShippingAddressElementOptions;
```

Reply:
Yes... I missed that. [https://github.com/stripe/stripe-js/pull/248](https://github.com/stripe/stripe-js/pull/248)

Reply:
Thank you!